src/pages/HomePage.vue

```diff
@@ -9,15 +9,30 @@
             exist. Yikes.
           </p>
         </div>
-        <BaseButton buttonType="large" caption="Book a ticket"/>
+        <BaseButton buttonType="primary" caption="Book a ticket"/>
    
       </div>
       <div class="main_image"></div>
     </section>
-
-    <section class="screening">
+    <div class="more-info">
+      <p>soon <span>in the cinema</span></p>
+      <p class="showItems" @click="changeVisible()">see all</p>
```

Comment:
We don't need to invoke a method in the template if it doesn't have any arguments. We are fine with passing just a function reference without parentheses - Vue will invoke it for us 😉